packages/next/pages/_document.js

```diff
@@ -50,7 +50,12 @@ export class Html extends Component {
 
   render() {
     const { amphtml } = this.context._documentProps
-    return <html amp={amphtml ? '' : null}>{this.props.children}</html>
+    const { children, ...props } = this.props
```

Comment:
nit: you can avoid this if you use a self closing tag:

```jsx
return <html {...this.props} amp={amphtml ? '' : null} />
```

Reply:
Let's do that in another PR 👌